exonum/src/blockchain/tests.rs

```diff
@@ -19,9 +19,9 @@ use std::collections::BTreeMap;
 use rand::{thread_rng, Rng};
 use serde_json;
 
-use blockchain::{Blockchain, Schema, Transaction};
+use blockchain::{Blockchain, Schema, Transaction, TransactionResult, TransactionValue};
 use crypto::{gen_keypair, Hash};
-use storage::{Database, Error, Fork, ListIndex};
+use storage::{Database, Fork, Error, ListIndex};
```

Comment:
Why? 🙃

Reply:
`rls` sorts entries in alphabetical order.

Reply:
It's not alphabetical order!

Reply:
Honestly, I have no idea. :smile:

Reply:
Rustfmt ?